resources/views/profile/two-factor-authentication-form.blade.php

```diff
@@ -120,7 +120,7 @@ class="button-secondary w-full sm:w-auto"
                 <div class="flex flex-col space-y-4 mt-8">
                     <x-ark-alert type="warning">
                         <x-slot name="message">
-                            @lang('fortify::pages.user-settings.2fa_warning_text')
+                            <span class="text-sm">@lang('fortify::pages.user-settings.2fa_warning_text')</span>
```

Comment:
are all alerts going to be `sm`? if so we should just change the alert component to match it instead of adding `sm` options

Reply:
@Highjhacker update to 2.5.1 of the UI lib, alerts are now `text-sm` by default